src/ui/FieldTable/FieldValue.ts

```diff
@@ -54,49 +56,75 @@ export class FieldValue extends Component {
    * @componentOptions
    */
   static options: IFieldValueOptions = {
+
     /**
-     * Specifies the field to be displayed by the FieldValue.<br/>
-     * This field is required.
+     * Specifies the field that the FieldValue should display.
+     *
+     * Specifying a value for this parameter is required in order for the FieldValue component to work.
      */
     field: ComponentOptions.buildFieldOption({ defaultValue: '@field', required: true }),
+
     /**
-     * Specifies the facet to be toggled when the component is clicked On.<br/>
-     * When no value is specified, the value of the <code>field</code> option is used.<br/>
-     * If the facet ID is custom (e.g. not the same name as its field), you must specify
-     * manually this option in order to link the correct facet.
+     * Specifies the {@link Facet} component to toggle when the end user clicks the FieldValue.
+     *
+     * Default value is the value of {@link FieldValue.options.field}.
+     *
+     * **Note:**
+     * > If the target {@link Facet.options.id} is is not the same as its {@link Facet.options.field}), you must specify
+     * > this option manually in order to link to the correct Facet.
      */
     facet: ComponentOptions.buildStringOption({ postProcessing: (value, options) => value || options.field }),
+
     /**
-     * Specifies if the content to display is an HTML element.<br/>
-     * The default value is <code>false</code>.
+     * Specifies whether the content to display is an HTML element.
+     *
+     * Default value is `false`.
      */
     htmlValue: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+
     /**
-     * Specifies if the field value is to be split at each {@link FieldValue.options.separator}.
-     * This is useful for splitting groups by a facet field.<br/>
-     * The values displayed are split by the {@link FieldValue.options.displaySeparator}.<br/>
-     * The default value is <code>false</code>.
+     * Specifies whether to split the FieldValue at each {@link FieldValue.options.separator}.
+     *
+     * This is useful for splitting groups using a {@link Facet.options.field}.
+     *
+     * When this option is `true`, the displayed values are split by the {@link FieldValue.options.displaySeparator}.
+     *
+     * Default value is `false`.
      */
     splitValues: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+
     /**
-     * Specifies the string used to split multi-value fields from the index.
-     * The default value is <code>;</code>.
+     * If {@link FieldValue.options.splitValues} is `true`, specifies the separator string which separates multi-value
+     * fields in the index.
+     *
+     * See {@link FieldValue.options.displaySeparator}.
+     *
+     * Default value is `";"`.
      */
-    separator: ComponentOptions.buildStringOption({ defaultValue: ';' }),
+    separator: ComponentOptions.buildStringOption({ depend: 'splitValues', defaultValue: ';' }),
```

Comment:
I made this depend of `splitValues` being `true` because it seems to make sense. Please make sure this is correct.